mon-pix/app/components/form-textfield.js

```diff
@@ -31,6 +31,12 @@ export default Component.extend({
   label: '',
   textfieldName: '',
   validationMessage: '',
+  isPassword: computed('textfieldName', function() {
```

Comment:
ici on ne return jamais false ? ça me paraît bizarre. je pense qu'on peut écrire : `return this.textfieldName === 'password'` directement.